doc/conf.py

```diff
@@ -207,7 +207,7 @@
 html_title = project
```

Comment:
Please try setting `html_title` to empty and see if the html_logo setting works.
```suggestion
html_title = ""
```